bmButtonHandlers.lua

```diff
@@ -150,8 +150,8 @@ function BMButtonHandlers.RenderButtonIcon(Button, cursorScreenPos, size)
 
     local draw_list = ImGui.GetWindowDrawList()
 
-    local iconId = Button.Icon
-    local iconType = Button.IconType
+    local iconId = Button.Icon or -1
+    local iconType = Button.IconType or ''

```

Comment:
This will change existing behavior (nil iconType means something) so please make sure this doesn't break existing stuff.